cli/terminal/table.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package terminal
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+	"text/tabwriter"
+)
+
+// PrintTable formats and prints a table.
+// +-----------+-----------+-----------+
+// | Header 1  | Header 2  | Header 3  |
+// +-----------+-----------+-----------+
+// | cell 11   | cell 12   | cell 13   |
+// | cell 21   | cell 22   | cell 23   |
+// +-----------+-----------+-----------+.
+func (t *term) PrintTable(header []string, body [][]string) {
```

Comment:
See previous comments regarding existing terminal UI and text table formatter packages.